addons/survey/static/src/img/burger_quiz_background.txt

```diff
@@ -0,0 +1,10 @@
+Picture by Haseeb Jamil published on Unsplash 
```

Comment:
Have to say I am curious with this ... is it something "standard" ? Do we have any guideline when using images with rights ?

Reply:
Ask @d-fence

Reply:
Hello there 👋 

I was checking something a semantical thing on the PR and I saw that  there was discussions about the image on the task chatter and on the PR.

This file is actually not needed, we can keep it if we want to but it's not mandatory.

> Even though **attribution isn’t required**, Unsplash photographers appreciate it as it provides exposure to their work and encourages them to continue sharing.

https://unsplash.com/license